### PR TITLE
e2e: add a template test with a lease

### PR DESCRIPTION
**Pull request by bbguimaraes**

### CODE REVIEW

File: test/e2e/lease/config.yaml [hunk 1]
```diff
@@ -3,17 +3,15 @@ base_images:
     name: centos
     namespace: openshift
     tag: '7'
-releases:
```
Comment (stevekuznetsov) [inline test/e2e/lease/config.yaml:6]: Ideally we'd have a case with both?

**Comment by bbguimaraes** [comment]
Updated to use https://github.com/openshift/ci-tools/pull/1267.

**Comment by stevekuznetsov** [comment]
/test e2e

```
   * could not initialize namespace: could not create role binding for: rolebindings.rbac.authorization.k8s.io is forbidden: User "system:serviceaccount:ci:ci-operator" cannot create resource "rolebindings" in API group "rbac.authorization.k8s.io" in the namespace "ci-op-426jmksd" 
```

**Comment by stevekuznetsov** [comment]
Part of DPTP-1365

**Pull request merged**